x-pack/platform/plugins/shared/streams/server/lib/streams/service.ts

```diff
@@ -41,12 +42,15 @@ export class StreamsService {
     request: KibanaRequest;
     assetClient: AssetClient;
   }): Promise<StreamsClient> {
-    const [coreStart] = await this.coreSetup.getStartServices();
+    const [coreStart, pluginStart] = await this.coreSetup.getStartServices();
 
     const logger = this.logger;
 
     const scopedClusterClient = coreStart.elasticsearch.client.asScoped(request);
 
+    const fakeRequest = createFakeRequestBoundToDefaultSpace(request);
+    const rulesClient = await pluginStart.alerting.getRulesClientWithRequest(fakeRequest);
```

Comment:
💬  We might find a better solution soon with the alerting team, using `space_id: "*"`